grassroots-backend/src/testing/PropsOf.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, expect, it } from "vitest";
+import { PropsOf } from "../grassroots-shared/util/PropsOf";
+
+const __brand: unique symbol = Symbol();
+
+class Branded<TBrand> {
+  readonly [__brand]!: TBrand;
+  next?: Branded<TBrand>;
+}
+
+describe("propsOf", () => {
+  it("should exclude brands", () => {
+    const x: PropsOf<Branded<"foo">> = { next: undefined };
+    expect(x.next).toStrictEqual(undefined);
```

Comment:
I'm not sure I understand this test. I would have expected `expect(x).toStrictEqual({next:undefined})`. Maybe I don't understand what's begins tested here.

Reply:
You're right! This is busted!

Fixed.